kafka_consumer/datadog_checks/kafka_consumer/kafka_consumer.py

```diff
@@ -123,7 +129,9 @@ def _create_kafka_admin_client(self):
             bootstrap_servers=kafka_connect_str,
             client_id='dd-agent',
             request_timeout_ms=self.init_config.get('kafka_timeout', DEFAULT_KAFKA_TIMEOUT) * 1000,
-            api_version=self.instance.get('kafka_client_api_version'),
+            # There is a bug with kafka-python where pinning api_version for KafkaAdminClient raises an
+            # `IncompatibleBrokerVersion`. Change to `api_version=self._kafka_version` once fixed upstream.
```

Comment:
is there an upstream issue/PR that can be linked here?

Reply:
upstream PR linked. PR is references related issues.